### ISSUE TITLE
Printing package directPrintPdf() doesn't work on iOS15

### ISSUE THREAD

**Issue by akkunchoi** [DavBfr/dart_pdf#823]
**Describe the bug**

When my app called `directPrintPdf()` on iOS15, it crashed and dumped the following error.
`layoutPdf()` is no problem.

```
*** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: 'Application tried to present modally a view controller <UIAlertController: 0x134038000> that is already being presented by <UIViewController: 0x133510420>.'
*** First throw call stack:
(0x181e3305c 0x19a34df54 0x18471a37c 0x1846813d8 0x1843b8064 0x184504a34 0x1844a0dfc 0x184617638 0x184529de0 0x18515e194 0x18515e62c 0x18515e218 0x18515ed3c 0x18516f35c 0x185168f98 0x181aa3c04 0x181aa5950 0x181ab3d30 0x181debce4 0x181da5ebc 0x181db93c8 0x19d5ca38c 0x18475f0bc 0x1844dcbe8 0x1048b9edc 0x104be9a24)
libc++abi: terminating with uncaught exception of type NSException
* thread #1, queue = 'com.apple.main-thread', stop reason = signal SIGABRT
    frame #0: 0x00000001b87219c4 libsystem_kernel.dylib`__pthread_kill + 8
libsystem_kernel.dylib`__pthread_kill:
->  0x1b87219c4 <+8>:  b.lo   0x1b87219e4               ; <+40>
    0x1b87219c8 <+12>: pacibsp
    0x1b87219cc <+16>: stp    x29, x30, [sp, #-0x10]!
    0x1b87219d0 <+20>: mov    x29, sp
Target 0: (Runner) stopped.
Lost connection to device.
```

**To Reproduce**
Code snippet to reproduce the behavior:
```dart
    Printing.directPrintPdf(
      printer: printer,
      onLayout: _buildPdf,
      format: PdfPageFormat.a4
    );
```

**Expected behavior**

**Screenshots**

**Flutter Doctor**

```
```

**Smartphone (please complete the following information):**
 - iOS15

**Additional context**

same problem?
https://developer.apple.com/forums/thread/689876

I don't know if it's relevant, `@MainActor` was added in latest docs.
https://developer.apple.com/documentation/uikit/uiprintinteractioncontroller?changes=latest_minor

**Comment by DavBfr** [comment]
Don't hesitate to file a Pull request when you fixed it.

**Comment by algibe** [comment]
This error has been solved with the new version of ios 15.2.1
